github-to-slack/functions/index.js

```diff
@@ -33,26 +33,26 @@ exports.githubWebhook = functions.https.onRequest((req, res) => {
       // The JSON body is automatically parsed by Cloud Functions so we re-stringify it.
       .update(JSON.stringify(req.body, null, 0))
       .digest('hex');
-  const expectedSignature = `${cipher}=${hmac}`;
+      const expectedSignature = `${cipher}=${hmac}`;
 
   // Check that the body of the request has been signed with the GitHub Secret.
   if (secureCompare(signature, expectedSignature)) {
```

Comment:
Please put the exceptional case in the if statement, not the normal one. By the time I get to the 'Your x-hub-signature\'s bad and you should feel bad!' line I don't remember why